optashift-employee-rostering-gwtui/src/main/java/org/optaplanner/openshift/employeerostering/gwtui/client/pages/employeeroster/AvailabilityLane.java

```diff
@@ -0,0 +1,41 @@
+/*
+ * Copyright (C) 2018 Red Hat, Inc. and/or its affiliates.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *       http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.optaplanner.openshift.employeerostering.gwtui.client.pages.employeeroster;
+
+import java.time.OffsetDateTime;
+import java.util.List;
+
+import org.optaplanner.openshift.employeerostering.gwtui.client.rostergrid.model.Lane;
+import org.optaplanner.openshift.employeerostering.gwtui.client.rostergrid.model.SubLane;
+import org.optaplanner.openshift.employeerostering.shared.employee.EmployeeAvailabilityState;
+
+public class AvailabilityLane extends Lane<OffsetDateTime> {
```

Comment:
I think that there should be one employee per lane perhaps?

Reply:
Can you explain an advantage of having Employee per lane versus Availability per lane? This is something a manager will be using, and I don't think they are interested in when a particular employee is off, but rather what employees are off. Plus if it was per employee, we will probably have a lot of empty lanes, making it harder to find out who is off.

Reply:
I'm not sure about the advantages, I just thought we were rewriting the old version of the Employee Roster using the HTML/CSS grid. If the UX team, @mbarkley and @ge0ffrey agree, we can certainly keep the Availability lanes. Did we already discussed that with them?

Reply:
It a discussion to be had; I can easily change it back to Employee Lanes; need to do a meetup on this.

Reply:
Not sure I even understand what "Availability per lane" even means. So far in all our discussions, we have talked about the "Employee Roster" screen being a view of the Spot Roster data but organized by employee. IMO we should stick with that for now, and discuss alternatives later.